x-pack/osquerybeat/beater/action_handler.go

```diff
@@ -68,7 +67,7 @@ func (a *actionHandler) Execute(ctx context.Context, req map[string]interface{})
 func (a *actionHandler) execute(ctx context.Context, req map[string]interface{}) (int, error) {
 	ac, err := action.FromMap(req)
 	if err != nil {
-		return 0, fmt.Errorf("%v: %w", err, ErrQueryExecution)
+		return 0, errors.Join(err, ErrQueryExecution)
```

Comment:
Since go1.20, it is valid to include multiple `%w` verbs with the result that the error is essentially the same as the product of `errors.Join`, so to retain the original output this can be `return 0, fmt.Errorf("%w: %w", err, ErrQueryExecution)`.

Reply:
Done